SasquatchMac/SasquatchMacObjC/AppDelegate.m

```diff
@@ -22,11 +21,24 @@ - (instancetype)init {
   [MSMobileCenter start:@"8649b73e-6df0-4985-a039-8ab1453d44f3"
            withServices:@[ [MSAnalytics class], [MSCrashes class], [MSPush class] ]];
   [MobileCenterProvider shared].mobileCenter = [[MobileCenterDelegateObjC alloc] init];
-  return self;
+  [self updateMobileCenterState];
```

Comment:
Well, i doubt that users will be using MC components in a views, it is just this particular case, testing MC, but never the less, there is a problem with timing: applicationDidFinishLaunching is called after viewDidLoad. What if users want to track some event in viewDidLoad and MC has not been started yet. One possible solution is to introduce on macOS some kind of delegate, but that is not one liner either.